packages/sign_in_with_apple/sign_in_with_apple/darwin/Classes/SignInWithAppleError.swift

```diff
@@ -107,8 +107,9 @@ public enum SignInWithAppleError {
                 errorCode = "authorization-error/notHandled"
             case .failed:
                 errorCode = "authorization-error/failed"
-#if os(iOS) && swift(>=5.5)
-            // new case since Xcode 13, but only on iOS SDK at the moment
+#if os(iOS) && swift(>=5.5) || os(macOS) && swift(>=5.5.1)
```

Comment:
```suggestion
#if (os(iOS) && swift(>=5.5)) || (os(macOS) && swift(>=5.5.1))
```

Just to make it clear in regards to priority?

Reply:
Sure, why not :)